CHANGELOG.md

```diff
@@ -4,8 +4,14 @@ Versions
 2018-11-DEV
 ----------
 
+* Downgrade Maven version to 3.2.1 for Java 6 buildbox (Maven 3.2.3+ uses HTTPS by default)
+
+2018-11-12
```

Comment:
Please remove this, part of `2018-11-DEV`

Reply:
OK. Which version should i put for this fix ?

Reply:
The `2018-11-DEV` section contains the on-going changes, it will be updated during the next release.

Reply:
OK i add my fix in this section then